indexer/classificator_loader.cpp

```diff
@@ -45,10 +45,16 @@ namespace classificator
     LOG(LDEBUG, ("Reading of classificator started"));
 
     Platform & p = GetPlatform();
+    StyleReader & reader = GetStyleReader();
 
-    ReadCommon(p.GetReader("classificator.txt"), p.GetReader("types.txt"));
+    for (auto style : { MapStyleLight, MapStyleDark, MapStyleClear })
+    {
+      reader.SetCurrentStyle(style);
 
-    drule::LoadRules();
+      ReadCommon(p.GetReader("classificator.txt"), p.GetReader("types.txt"));
```

Comment:
Вообще-то это замедляет старт, особенно на дешевых андроидах.

> On 18.4.2016, at 11.50, ygorshenin <notifications@github.com> wrote:
> 
> In indexer/classificator_loader.cpp:
> 
> >  
> > -    drule::LoadRules();
> > +      ReadCommon(p.GetReader("classificator.txt"), p.GetReader("types.txt"));
> Это ок что мы 3 раза читаем одни и те же файлы, просто в разном контексте? Это не блокер, но нельзя ли их прочесть хотя бы один раз?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>